modules/polaris/CHANGELOG.md

```diff
@@ -0,0 +1,11 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
+and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
+
+## 4.0.8 - 2021-07-02
+
+### Improved
+
+- Improved detection of custom URL schemes #TINY-5074
```

Comment:
```suggestion
### Fixed
- Fixed detection of custom URL schemes #TINY-5074
```
This needs to be `Fixed` if it's going to be a patch release. We also shouldn't have the blank line.

Reply:
```suggestion
### Fixed
- Fixed incorrect `Regexes.link` URL detection for custom schemes #TINY-5074
```